packages/db/package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "prisma studio --port 5556",
     "clean": "rm -rf .turbo node_modules",
     "db-push": "prisma db push",
-    "db-generate": "prisma generate"
+    "db-generate": "prisma generate && ./fix-prisma.sh"
```

Comment:
Is this the 'proper way' to run this? I saw Rox's solution saying he added something to the `turbo.json`

https://github.com/prisma/prisma/issues/10433#issuecomment-1264687986